Makefile

```diff
@@ -46,6 +46,13 @@ be:
 	@docker-compose $(DC_FILE) build --force-rm --no-cache --parallel backend
 	@docker-compose $(DC_FILE) up -d --build backend
 
+cypress-keycloak:
```

Comment:
These can be improved, but I thought we could use some commands to deal with cypress.